packages/core/src/components/select/select.scss

```diff
@@ -0,0 +1,92 @@
+@use '../../helpers';
+@use '../../mixins';
+@use '../icon/icon';
+
+@mixin Select() {
+  @include icon.Icon();
+
+  @if not mixins.includes('Select') {
+    @include _Select();
+  }
+}
+
+@mixin _Select() {
+  .ods-select {
+    align-items: center;
+    display: inline-flex;
+    position: relative;
+    width: helpers.space(50);
+
+    &.-borderless {
+      width: fit-content;
+    }
+  }
+
+  .ods-select-native {
+    @include mixins.as-text-field();
+    @include mixins.interactive();
+    @include with-padding(1px); // border width set on `as-text-field` mixin
+
+    appearance: none; // removes native arrow that opens the select
+
+    &.-empty {
+      color: helpers.color('content-placeholder');
+    }
+
+    + .ods-icon {
+      color: helpers.color('content-main');
+      pointer-events: none;
+      position: absolute;
+      right: helpers.space(2);
+    }
```

Comment:
Wouldn't this icon be the same in the non-native version?

Reply:
Yes, but ignore non-native or "custom" at all for this.

Reply:
Cool. I was thinking like `.ods-select > .ods-icon` but no need to change anything now.